Nos/Assets/Colors.xcassets/image-background.colorset/Contents.json

```diff
@@ -0,0 +1,20 @@
+{
+  "colors" : [
```

Comment:
I think image-viewer-background would be more self-descriptive

Reply:
done!